src/openscad.cc

```diff
@@ -572,13 +606,16 @@ int cmdline(const CommandLine& cmd)
     .camera = cmd.camera,
   };
 
-  if (cmd.animate_frames == 0) {
+  if (cmd.animate.frames == 0) {
     render_variables.time = 0;
     return do_export(cmd, render_variables, export_format, root_file);
   } else {
     // export the requested number of animated frames
-    for (unsigned frame = 0; frame < cmd.animate_frames; ++frame) {
-      render_variables.time = frame * (1.0 / cmd.animate_frames);
+    for (unsigned frame = 0; frame < cmd.animate.frames; ++frame) {
+      if (frame % cmd.animate.num_shards != cmd.animate.shard) {
```

Comment:
Interesting approach to sharding. Intuitively, I'd think the most appropriate sharding approach would be to evaluate consecutive frames on the same node, as that may yield more coherent geometry and potentially more cache hits.
Did you consider the different approaches?

Reply:
No particular reason. In absence of other considerations (and I didn't know if caches made a big impact), I just defaulted to the default mod sharding as it usually allows for the nicest distribution with the least likelyhood to have some straggler shards.

I've played with range sharding now and it tends to be slightly better (only measured on my laptop, so results are probably more noisy). Maybe that is indeed the right approach, and it might also be easier to understand for the casual user. Will change it to range sharding.